apps/api/src/validation/dynamic-data-type.module.ts

```diff
@@ -16,7 +16,7 @@ const EmptyDtoProvider = {
 export class DynamicDataTypeModule implements OnApplicationBootstrap {
     constructor(private readonly dataTypeFinderService: DynamicDataTypeFinderService) {}
 
-    async onApplicationBootstrap() {
+    async onModuleInit() {
```

Comment:
It's not clear to me why TypeScript wanted this change. I don't think the interface was updated on Nest's end. I'll look into this.

Reply:
Fixed.